### PR TITLE
Amazon linux and webroot authenticator support

### PR DESCRIPTION
**Pull request by pataiji**
Included changes
- Fix deprecated warning --standalone_supported_challenges
- Amazon Linux support
  > ERROR :         stdout | FATAL: Amazon Linux support is very experimental at present...
ERROR :         stdout | if you would like to work on improving it, please ensure you have backups
ERROR :         stdout | and then run this script again with the --debug flag!
ERROR :         Command `/usr/bin/certbot-auto -n --os-packages-only` failed. (exit status: 1)
ERROR :       execute[install dependency package] Failed.
- webroot authenticator support
- only execute resources when needed

### CODE REVIEW

File: lib/itamae/plugin/recipe/letsencrypt/get.rb [hunk 1]
```diff
@@ -4,26 +4,46 @@
...
+      '--agree-tos',
+      "-d #{domain}",
+      "-m #{node[:letsencrypt][:email]}",
+      "-a #{node[:letsencrypt][:authenticator]}",
+      '--keep',
+      '-n',
+      "--preferred-challenges #{node[:letsencrypt][:challenge_type]}",
+    ]
+    cmd << "-w #{node[:letsencrypt][:webroot_path]}" if node[:letsencrypt][:webroot_path]
+    cmd << '--debug' if node[:platform] == 'amazon'
```
Comment (hatappi) [inline lib/itamae/plugin/recipe/letsencrypt/get.rb]: debug option can be used on other platforms.
How about adding `debug_mode` to node.yml ?

**Comment by hatappi** [comment]
@pataiji 
Thank you for PR.
I added comment, please check it.

**Comment by pataiji** [comment]
@hatappi 
Thanks for responding.
I have added debug_mode!
Please review it again.

**Comment by hatappi** [comment]
Thank you for this PR.

**Pull request merged**